src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.simplefileshare.simplefileshare.controller;
+import org.springframework.web.bind.annotation.*;
+import java.io.File;
+
+@RestController
+@RequestMapping("/api/upload")
+public class FileController {
+
+    @GetMapping("/api/get")
```

Comment:
No need to give endpoint here if we are using Rest mappings and giving an endpoint on the class itself

Reply:
Yes added the comment in it not needed already :p